### PR TITLE
feat: add members count to sigs

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Comment by hi-rustin** [comment]
@Mini256  PTAL~ Thanks!

File: src/repositoies/sig/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      .createQueryBuilder()
+      .select("*")
+      .from((sub) => {
+        return sub
+          .select("s.id, count(sm.contributor_id) as membersCount")
+          .from(Sig, "s")
+          .leftJoin(SigMember, "sm", "sm.sig_id = s.id")
+          .where(`s.status = ${publicSigStatus}`)
+          .groupBy("s.id");
+      }, "sigs")
```
Comment (Mini256) [inline src/repositoies/sig/index.ts:32]: I think modifying the alias of the internal table to `sig_summary` and the field `s.id` of this table alias to `sig_id` will be better to understand.

File: test/services/sig/index.test.ts [hunk 2]
```diff
@@ -462,36 +464,30 @@ describe("Sig Service", () => {
...
+        needsLGTM: 2,
       },
     ];
-    const sigFindOneMock = jest.spyOn(sigRepository, "findAndCount");
-    sigFindOneMock.mockImplementation((options) => {
-      // @ts-ignore
-      const { skip, take } = options;
 
-      return Promise.resolve([sigs.slice(skip, take), sigs.length]);
+    const sigFindOneMock = jest.spyOn(sigRepository, "listSigsAndCount");
```
Comment (Mini256) [inline test/services/sig/index.test.ts:488]: There seems to be an inconsistency between the variable name and the function of mock.

**Review by Mini256 — CHANGES_REQUESTED** [review]

File: test/services/sig/index.test.ts (hunk 2 above)
Comment (hi-rustin) [reply test/services/sig/index.test.ts]: Fixed.

File: src/repositoies/sig/index.ts (hunk 1 above)
Comment (hi-rustin) [reply src/repositoies/sig/index.ts]: Renamed.

File: src/repositoies/sig/index.ts [hunk 3]
```diff
@@ -29,9 +29,9 @@ export default class SigRepository extends Repository<Sig> {
           .leftJoin(SigMember, "sm", "sm.sig_id = s.id")
           .where(`s.status = ${publicSigStatus}`)
           .groupBy("s.id");
-      }, "sigs")
-      .leftJoin(Sig, "sig", "sig.id = sigs.id")
-      .orderBy("membersCount", "DESC")
+      }, "sig_summary")
+      .leftJoin(Sig, "sig", "sig.id = sig_summary.id")
```
Comment (Mini256) [inline src/repositoies/sig/index.ts:33]: select("s.id sig_id, count(sm.contributor_id) as membersCount")

sig.id = sig_summary.sig_id

**Pull request merged**